### DISCUSSION TITLE
Client ID cannot be identified over DoT/DoQ

### DISCUSSION THREAD

**Discussion by sekac55** [AdguardTeam/AdGuardHome#3903] [Q&A]
Encryption settings setup:
Status:
Certificate chain is valid
Subject: CN=example.com
Issuer: CN=R3,O=Let's Encrypt,C=US
Expires: 2022-02-28 09:38:02
Hostnames: *.example.de, example.de (Wildcard certificate is there)

The TLS and QUIC working without issues. Only with the Client ID its not working.

https://github.com/AdguardTeam/AdGuardHome/wiki/Clients#idclient

DNS-over-TLS: tls://my-client.example.org (requires a Wildcard certificate)
DNS-over-QUIC: quic://my-client.example.org (requires a Wildcard certificate)

Can you help me?

**Comment by ainar-g** [comment]
You can use the [`dnslookup`][1] utility to try and make a request and see what the problem is:

```sh
dnslookup 'example.com' 'tls://my-client.example.de'
```

[1]: https://github.com/ameshkov/dnslookup

**Comment by EugeneOne1** [comment]
@sekac55, may be related to #3914.  Could you please install the latest edge build and check if the behavior remains?

**Comment by sekac55** [comment]
I have installed the Version: v0.107.0-a.218+90e65b66
Same issue. No internet connection with client id over QUIC/TLS.  DoH works like a charm with client id.
Im running Ubuntu 20.4 on VPS Server with UFW. Ports: 22, 443, 784 and 853 are open.
